Beat Saber Utils/Utilities/BSEvents.cs

```diff
@@ -144,15 +144,15 @@ private void OnMenuSceneWasLoadedFresh(ScenesTransitionSetupDataSO transitionSet
             gameScenesManager.transitionDidFinishEvent -= OnMenuSceneWasLoadedFresh;
 
             var levelDetailViewController = Resources.FindObjectsOfTypeAll<StandardLevelDetailViewController>().FirstOrDefault();
-            levelDetailViewController.didChangeDifficultyBeatmapEvent += delegate (StandardLevelDetailViewController vc, IDifficultyBeatmap beatmap) { InvokeAll(difficultySelected, vc, beatmap); };
+            levelDetailViewController.didChangeDifficultyBeatmapEvent += delegate (StandardLevelDetailViewController vc) { InvokeAll(difficultySelected, vc, vc.beatmapKey); };
```

Comment:
I think that keeping the original signature of the event is better... We can access the beatmap key manually if needed.